Code/GraphMol/MolDraw2D/MolDraw2D.h

```diff
@@ -79,12 +91,26 @@ class MolDraw2D {
       const std::map<int, double> *highlight_radii = NULL, int confId = -1);
 
   virtual void drawMolecule(
+      const ROMol &mol, const std::string &legend,
+      const std::vector<int> *highlight_atoms = NULL,
+      const std::map<int, DrawColour> *highlight_map = NULL,
+      const std::map<int, double> *highlight_radii = NULL, int confId = -1);
+
+  virtual void drawMolecule(
```

Comment:
Exactly: it's to provide a decent API while maintaining backwards compatibility. I agree completely that it's a lot of overloads, but this is one of those situations where C++'s lack of real keyword arguments is a major pain in the ass.